src/averageAges.js

```diff
@@ -14,7 +14,28 @@
  *
  * @return {number}
  */
+
+const sexMale = 'm';
+const sexFemale = 'f';
+
+function calculateAge(array) {
```

Comment:
You don't just count the age, but return the `average` value, so you need to add the name more precisely. 
Also, don't use an `array` to specify an argument, choose a more meaningful name.